packages/oraft/oraft.0.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "komamitsu@gmail.com"
+authors: ["Mitsunori Komatsu"]
+homepage: "https://github.com/komamitsu/oraft"
+bug-reports: "https://github.com/komamitsu/oraft/issues"
+dev-repo: "git+https://github.com/komamitsu/oraft.git"
+description: "Raft consensus algorithm implemented in OCaml"
+synopsis: "Raft consensus algorithm implemented in OCaml"
+license: "Apache"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+    "dune" {build}
+    "core"
```

Comment:
Something tells me this is going to require a version constraint somewhere. This library changed in API often in the past so it's a good idea to know which version is required at a minimum (probably something like `>= "v0.9.0"`)